agent/src/main/resources/application-test.yml

```diff
@@ -0,0 +1,19 @@
+app:
+  device-script:
+    commands:
+      - type: ADBShell
+        when: setUp
+        matcher: '.*'
+        inline: |
+          test command 1
+      - type: ADBShell
+        when: setUp
+        matcher: '.*test.*'
+        inline: |
+          test command 2
+          test command 3
+      - type: ADBShell
+        when: tearDown
+        matcher: 'com.microsoft.test'
```

Comment:
If it's a matcher for app name, do you think we can name it as packageNameMatcher?

Reply:
Or matcher:
          package-name: 'com.microsoft.test'

Reply:
Actually, the matcher is used to match the suite Class name. I will rename it as suite-class-matcher and add a comment: If the suite-class doesn't be provided, the value of suite-class would be the package name of app.